### PR TITLE
Generate documentation report 

### PR DESCRIPTION
**Pull request by andrii-balitskyi**
https://github.com/seamapi/docs/issues/364

### CODE REVIEW

File: docs/api/_report.md [hunk 1]
```diff
@@ -0,0 +1,368 @@
...
+
+### Endpoints
+
+
+### Endpoint parameters
+
+
+## Missing `return_type`
+
+These routes are missing a `return_type` and should be fixed.
```
Comment (razor-x) [inline docs/api/_report.md:341]: I meant to say `response_key`. Actually, we may not even need this section once we merge https://github.com/seamapi/blueprint/pull/109

Comment (andrii-balitskyi) [reply docs/api/_report.md:341]: 21444ff49a7a756d74a7f69e3f3a0b9d31e61847

**Comment by andrii-balitskyi** [comment]
These https://github.com/seamapi/blueprint/pull/116, https://github.com/seamapi/blueprint/pull/117 are required to fix the build

File: docs/api/_report.md [hunk 2]
```diff
@@ -0,0 +1,514 @@
...
+- `thermostat_schedule.ends_at`
+- `thermostat_schedule.max_override_period_minutes`
+- `thermostat_schedule.name`
+- `thermostat_schedule.starts_at`
+- `thermostat_schedule.thermostat_schedule_id`
+
+### Endpoints
+
+- `/acs/access_groups/add_user`
+- `/acs/access_groups/add_user`
```
Comment (DebbieAtSeam) [inline docs/api/_report.md:136]: Why are some of these repeated?

File: docs/api/_report.md [hunk 3]
```diff
@@ -0,0 +1,514 @@
...
+- `/acs/users/create`
+    - `email`: use email_address.
+- `/acs/users/update`
+    - `email`: use email_address.
+- `/acs/users/update`
+    - `email`: use email_address.
+- `/thermostats/set_fan_mode`
+    - `fan_mode`: use fan_mode_setting instead.
+
+### Extra response keys
```
Comment (DebbieAtSeam) [inline docs/api/_report.md:441]: What does this mean?

**Review by DebbieAtSeam — APPROVED** [review]
@andrii-balitskyi The report and added code sample language are excellent!!! Thanks! 

@razor-x Do we need to wait to merge this PR, or will it not affect the open GitBook CRs?

**Comment by razor-x** [comment]
@DebbieAtSeam I'll isolate the doc changes in another PR so this one will only be the report. But I don't expect any merge conflicts for PRs that only change generated files.

File: docs/api/_report.md (hunk 3 above)
Comment (razor-x) [reply docs/api/_report.md:441]: Some endpoints return more than one top-level key (one that is not the `response_key`). Anything in this list is something we would want to eventually remove, or for new endpoints, would indicate a mistake.

File: docs/api/_report.md (hunk 2 above)
Comment (andrii-balitskyi) [reply docs/api/_report.md:136]: Thanks for catching this! It turns out that our blueprint defines an endpoint for each method that the endpoint supports. So, in the case of `/acs/access_groups/add_user`, there's one endpoint for `PUT` and one for `POST`. @razor-x, is this the intended design for the blueprint?
Anyway, I'll deduplicate the endpoints for now.

Comment (andrii-balitskyi) [reply docs/api/_report.md]: Deduped endpoint here b29e2ee66a695d562e447513e756882ce4218ed9

Comment (razor-x) [reply docs/api/_report.md]: No, this is a bug in blueprint then. There should be one endpoint per path, the supported methods defined on the endpoint

Comment (razor-x) [reply docs/api/_report.md]: https://github.com/seamapi/blueprint/issues/121

**Review by razor-x — APPROVED** [review]

**Comment by razor-x** [comment]
Nice work @andrii-balitskyi ! Bringing this in

**Pull request merged**